2020-11-12T17:51:45.6655468+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-12T17:51:45.7148918+02:00  [INF] Now listening on: "http://localhost:5002" (d826f4b8)
2020-11-12T17:51:45.7210244+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-12T17:51:45.7242092+02:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-12T17:51:45.7266163+02:00  [INF] Content root path: "J:\Applications\SuperStore\src\Web\WebApi" (b5d60022)
2020-11-12T17:55:24.8007426+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-12T17:55:24.8097791+02:00  [INF] Now listening on: "http://localhost:5002" (d826f4b8)
2020-11-12T17:55:24.8120642+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-12T17:55:24.8146471+02:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-12T17:55:24.8167358+02:00  [INF] Content root path: "J:\Applications\SuperStore\src\Web\WebApi" (b5d60022)
2020-11-12T17:55:26.1890682+02:00 0HM474DF73IEJ:00000001 [ERR] An unhandled exception has occurred while executing the request. (48a46595)
System.InvalidOperationException: Unable to resolve service for type 'Microsoft.Extensions.Logging.ILogger' while attempting to activate 'WebApi.Controllers.ProductController'.
   at Microsoft.Extensions.DependencyInjection.ActivatorUtilities.GetService(IServiceProvider sp, Type type, Type requiredBy, Boolean isDefaultParameterRequired)
   at lambda_method(Closure , IServiceProvider , Object[] )
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerActivatorProvider.<>c__DisplayClass4_0.<CreateActivator>b__0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Controllers.ControllerFactoryProvider.<>c__DisplayClass5_0.<CreateControllerFactory>g__CreateController|0(ControllerContext controllerContext)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2020-11-12T17:56:36.7226242+02:00  [INF] Now listening on: "https://localhost:5001" (d826f4b8)
2020-11-12T17:56:36.7532722+02:00  [INF] Now listening on: "http://localhost:5002" (d826f4b8)
2020-11-12T17:56:36.7603976+02:00  [INF] Application started. Press Ctrl+C to shut down. (dcaefe54)
2020-11-12T17:56:37.1629295+02:00  [INF] Hosting environment: "Development" (c3307c92)
2020-11-12T17:56:37.2541956+02:00  [INF] Content root path: "J:\Applications\SuperStore\src\Web\WebApi" (b5d60022)
